### ISSUE TITLE
Instalation fault  on windows 11 con mamba o Conda

### ISSUE THREAD

**Issue by crn565** [nilmtk/nilmtk-contrib#87]
I am trying to install nilmtk_env using anaconca by the recommended method**conda create -n nilm -c conda-forge -c nilmtk nilmtk-contrib** and not works ( black screen un hours and not responding)

I've tried to install with Mamba . First I create the enviremente, I install de dependecies( keras,cvxpy and nilmtk) and finally I try to install:

- mamba create -n nilmtk-contrib-env python=3.8
- mamba activate nilmtk-contrib-env
- mamba install -c conda-forge numpy pandas matplotlib scipy scikit-learn jupyter
- mamba install -c conda-forge cvxp
- mamba install -c conda-forge  keras
- mamba install -c conda-forge  nilmtk
- git clone https://github.com/nilmtk/nilmtk-contrib.git
- cd nilmtk-contrib
- pip install -e .

And in the last commnad I received and error that tell that nilmtk version must be >=0.4  but  my version of nilmtk is 0.4.3 so I don't understand the error  the instalation..

I don't know how to do to install nilmtk-contrib ! please help!

**Comment by crn565** [comment]
**And by the way this is my installed version of NILMTK:**

(nilmtk-contrib-env) C:\Users\carlo\nilmtk-contrib>mamba list |findstr nilmtk
# packages in environment at C:\Users\carlo\miniforge3\envs\nilmtk-contrib-env:
nilm_metadata             0.2.4                         0    nilmtk
nilmtk                    0.4.3                      py_0    nilmtk

(nilmtk-contrib-env) C:\Users\carlo\nilmtk-contrib>

**Comment by olarraUPC** [comment]
@crn565 did you manage to install it? I'm still having issues

**Comment by crn565** [comment]
Yes , I unsinstalled anaconda and resinstall  with conda create -n nilm -c conda-forge -c nilmtk nilmtk-contrib
The  paquet nilmtk_contrib is intalled but when I used it not works

**Comment by khirds** [comment]
@crn565  any Updates ,is NILMtk working .Please guide?

**Comment by crn565** [comment]
Thank you, it works (with modifications). I followed the following manual https://soloelectronicos.com/2025/04/04/solucionar-problemas-instalando-nilmtk-contrib/

**Comment by khirds** [comment]
@crn565  do you have any git account for modified version ?
